components/notability/wikis/ageofempires/notability_checker_config.lua

```diff
@@ -210,7 +210,7 @@ end
 -- want to decrease the points given for a certain mode
 function Config.adjustScoreForMode(score, mode)
 	local modeMod = 1
-	if mode == "2v2" or "3v3" or "4v4" then
+	if mode == '2v2' or '3v3' or '4v4' then
```

Comment:
this line doesn't make sense
should probably open an issue to fix it
(out of scope for the pr)

Reply:
Whoops, #2258

Reply:
@mbergen could you do me a fav and resolve the conflict in this pr?
thx in advance

Reply:
Sure, done